packages/app/src/systems/Chains/eth/services/assetService.tsx

```diff
@@ -0,0 +1,48 @@
+import { isAddress } from 'viem';
+
+import type { BridgeAsset } from '~/systems/Bridge';
+import { db } from '~/systems/Core/utils/database';
+
+export class AssetService {
```

Comment:
like in other services, create and export  Inputs to help centralize and keep consistency